paasta_tools/paasta_cli/schemas/chronos_schema.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "http://paasta.readthedocs.org/en/latest/yelpsoa_configs.html#chronos-clustername-yaml",
+    "type": "object",
+    "minProperties": 1,
+    "additionalProperties": {
+        "type": "object",
+        "properties": {},
```

Comment:
I stole the schemas from our existing Gitolite checks. I am fine modifying them, but I would like to do that as a separate issue after this one is resolved.